app/layout.tsx

```diff
@@ -36,6 +36,7 @@ export default function RootLayout({
   return (
     <html lang="en">
       <body className={`${GTFlexa.variable} antialiased`}>
+        <GridHelper />
```

Comment:
I think we need a check if we are in develop environment here. We do not want to include this in our hosting environments.